spec/requests/api/v1/item_request_spec.rb

```diff
@@ -53,7 +64,7 @@
     expect(response).to be_success
     expect(response_item.first["name"]).to eq(item.name)
     expect(response_item.first["description"]).to eq(item.description)
-    expect(response_item.first["unit_price"]).to eq(item.unit_price)
+    expect(response_item.first["unit_price"]).to eq("0.15")
```

Comment:
I don't think so for our tests, but it's something to keep in mind if the tests break in the future. I think we have to keep it there for the serializer because that's how it's returned from the request